src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/Transaction.java

```diff
@@ -10,6 +10,8 @@
     private final ReferenceTransactionNumber referenceTransactionNumber =
         ReferenceTransactionNumber.fromString(extractSegment(ReferenceTransactionNumber.KEY_QUALIFIER));
 
+    // TO-DO insert segments
```

Comment:
"TO-DO" as part of this story, or do we know which story will fulfil it and can indicate it here?

Reply:
I've removed the TO-DO for now as after the conversation this morning, this may be renamed or even removed in the upcoming stories